src/parcels/mount-parcel.ts

```diff
@@ -25,10 +23,9 @@ import { toUpdatePromise } from "../lifecycles/update";
 import { toUnmountPromise } from "../lifecycles/unmount";
 import { ensureValidAppTimeouts } from "../applications/timeouts";
 import { formatErrorMessage } from "../applications/app-errors";
-import { isParcel } from "../applications/app.helpers";
 
 let parcelCount = 0;
-const rootParcels = { parcels: {} };
+const rootParcels: ParcelOwner = { name: "rootParcels", parcels: {} };
```

Comment:
Adding the `name` property is a runtime change to facilitate implementing the `ParcelOwner` interface.